### DISCUSSION TITLE
Show nutrition values by default on v1?

### DISCUSSION THREAD

**Discussion by jeffxt** [mealie-recipes/mealie#3324] [Q&A] [answered]
I'm currently migrating from Mealie pre-v1.0 to the new v1.0.  However, I noticed that all of my recipes don't have any nutrition values when I did an import of my existing recipes.  I have to manually turn them on:

Edit > Settings > enable `Show Nutrition Values`

Can this setting be enabled by default?  I tried looking in the settings but maybe I'm missing something?

**Comment by michael-genson** [comment, marked as answer]
There's a setting for new recipes to enable nutrition values by default in your group settings (click on your profile -> manage group)

To bulk-enable nutrition settings on all your existing recipes, you can go to the data management pages (click on your profile -> manage data -> choose recipes from the drop down -> actions -> update settings)

**Reply by jeffxt** [reply]
Perfect, that worked!  I'm still learning the v1 changes, so thanks for being patient with me and appreciate the reply! :)
